prow/crier/reporters/gerrit/reporter.go

```diff
@@ -594,6 +599,13 @@ func GenerateReport(pjs []*v1.ProwJob, customCommentSizeLimit int) JobReport {
 	linesWithURLs := make([]string, len(report.Jobs))
 	linesWithoutURLs := make([]string, len(report.Jobs))
 	for i, job := range report.Jobs {
+		// If the URL cannot be found (for example, because the job has not been
+		// scheduled due to some other failure), then use the string
+		// URL_NOT_FOUND in its place instead. This way, we don't report jobs
+		// with an empty URL field.
+		if job.URL == "" {
```

Comment:
would also expect a unit test that produces comment in this style

Reply:
Isn't that covered in the test case I added?

Reply:
oh sorry I read it wrong, thought it's for parsing